crates/typos-cli/src/default_types.rs

```diff
@@ -315,6 +315,7 @@ pub(crate) const DEFAULT_TYPES: &[(&str, &[&str])] = &[
     ("ts", &["*.ts", "*.tsx", "*.cts", "*.mts"]),
     ("twig", &["*.twig"]),
     ("txt", &["*.txt"]),
+    ("typ", &["*.typ"]),
```

Comment:
This file is copied out of ripgrep
https://github.com/BurntSushi/ripgrep/blob/master/crates/ignore/src/default_types.rs

Changes to it could easily b lost when we do updates from ripgrep

Reply:
It looks like this will make it into ripgrep in some time. It's currently staged in BurntSushi's `ag/rollup` branch following <https://github.com/BurntSushi/ripgrep/pull/2914>: <https://github.com/BurntSushi/ripgrep/blob/ag/rollup/crates/ignore/src/default_types.rs#L302>

Reply:
#1396 updates the type list from ripgrep